editor/src/components/inspector/sections/header-section/header-section.tsx

```diff
@@ -22,9 +20,9 @@ export interface HeaderSectionCoreProps extends ElementPathProps {
 }
 export interface HeaderSectionProps
   extends HeaderSectionCoreProps,
-    NameRowInnerProps,
-    LayoutWrapperCoreProps,
-    LayoutWrapperRowProps {}
+  NameRowInnerProps,
```

Comment:
this is weird. was the previous code not formatted or is the new code not formatted by Prettier? or is this a prettier version change?